kotlin-reference-server/src/main/kotlin/org/stellar/reference/plugins/ConfigureAuth.kt

```diff
@@ -0,0 +1,40 @@
+package org.stellar.reference.plugins
+
+import com.auth0.jwt.JWT
+import com.auth0.jwt.algorithms.Algorithm
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.auth.*
+import io.ktor.server.auth.jwt.*
+import io.ktor.server.response.*
+import org.stellar.reference.data.AuthSettings
+import org.stellar.reference.data.Config
+
+fun Application.configureAuth(cfg: Config) {
+  when (cfg.authSettings.type) {
+    AuthSettings.Type.JWT ->
+      authentication {
+        jwt("integration-auth") {
```

Comment:
❓ Should we name this something different considering we'd use the same auth type in our persistent deployment eventually?

Reply:
Change to `endpoint-auth`.